### ISSUE TITLE
Recommended way to handle os.Exit()?

### ISSUE THREAD

**Issue by HTechHQ** [tommy-muehle/go-mnd#5]
Hello,
just updated golangci-lint and got the following error:

```
main.go:37:11: mnd: Magic number: 1, in <argument> detected (gomnd)
                os.Exit(1)
```

This seams a bit strict to me. How would you recommend handling this situation?
Not sure if it's actually a linter issue or I should update my code.

Thx

**Comment by tommy-muehle** [comment]
I'll also exclude 1 by default because there are more false-positives as real scenarios.

**Comment by tommy-muehle** [comment]
Fixed with [v2.0.0](https://github.com/tommy-muehle/go-mnd/releases/tag/v2.0.0).
